src/UI/Root/Components/Header/EnvSelector/EnvSelector.tsx

```diff
@@ -44,11 +44,11 @@ export const EnvSelector: React.FC<Props> = ({
           aria-label={toggleText}
           isFullHeight
           onClick={() => setIsOpen(!isOpen)}
-          icon={!authHelper.isDisabled() ? <UserCircleIcon /> : null}
+          icon={!authHelper.getUser() ? <UserCircleIcon /> : null}
         >
           <StyledDiv>
             <div>
-              {!authHelper.isDisabled() && (
+              {!authHelper.getUser() && (
```

Comment:
checking if you don't have a user to then call the user, doesn't make sense at all.